lib/t_web/channels/profile_channel.ex

```diff
@@ -107,12 +107,15 @@ defmodule TWeb.ProfileChannel do
   end
 
   defp replace_photo_url_with_s3key(%{"s3_key" => _} = background), do: background
-  defp replace_photo_url_with_s3key(%{"color" => _} = bg), do: bg
 
-  defp replace_photo_url_with_s3key(%{"proxy" => proxy_url}) do
-    %{"s3_key" => s3_key_from_proxy_url(proxy_url)}
+  defp replace_photo_url_with_s3key(%{"proxy" => proxy_url} = background) do
```

Comment:
what's in `background`?

Reply:
e.g. https://github.com/getsince/test3/pull/693/files#diff-4b57c8e03ec7e4456f02252bb310d0189cf01a8285d6e238c9698a5a78c2bc57R357